internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
```

Comment:
As the structs grow, consider adding methods to validate or format the data associated with each type. This can help ensure that the data integrity remains intact as the application scales.